stable/rabbitmq-ha/README.md

```diff
@@ -138,6 +138,7 @@ and their default values.
 | `rbac.create`                                  | If true, create & use RBAC resources                                                                                                                                                                  | `true`                                                     |
 | `replicaCount`                                 | Number of replica                                                                                                                                                                                     | `3`                                                        |
 | `resources`                                    | CPU/Memory resource requests/limits                                                                                                                                                                   | `{}`                                                       |
+| `securityContext.fsGroup`                      | Group ID for the container, mitigates permission errors on a cluster with `RunAsGroup=false`                                                                                                          | `101`                                                      |
```

Comment:
There's no RunAsGroup=false. Can you remove the text `, mitigates permission errors on a cluster with RunAsGroup=false`?

Reply:
Oh, you meant the feature gate named that. That actually doesn't impact your problem. You need fsGroup regardless.

Reply:
Sure, will do.

Reply:
Done. I've changed the field's description to _"Group ID for the container's volumes"_ so one can differentiate it from the `runAsGroup`'s description.

Reply:
Perfect, thanks